pgaudit-11.yaml

```diff
@@ -0,0 +1,38 @@
+# pgAudit v1.3.X is intended to support PostgreSQL 11.
+# https://github.com/pgaudit/pgaudit/blob/master/README.md#postgresql-version-compatibility
+package:
+  name: pgaudit-11
+  version: 1.3.4
+  epoch: 0
+  description: PostgreSQL Audit Extension
+  copyright:
+    - license: BSD
```

Comment:
License looks fine. is actually 'PostgreSQL License' but I see they note its to be interrupted similar to BSD, so this is okay.